docs/guides/guide-delete-or-replace-metrics.md

```diff
@@ -0,0 +1,299 @@
+
+### Precondition
+
+- [Single-node VictoriaMetrics](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html);
+- [Cluster version of VictoriaMetrics](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html);
+- [curl](https://curl.se/docs/manual.html)
+- [jq tool](https://stedolan.github.io/jq/)
+
+## How to delete metrics
+
+_Warning: time series deletion is not recommended to use on a regular basis. Each call to delete API could have a performance penalty. The API was provided for one-off operations for deleting malformed data or tp satisfy GDPR compliance._
+
+[Delete API](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-delete-time-series) expects from user to specify [time series selector](https://prometheus.io/docs/prometheus/latest/querying/basics/#time-series-selectors). So the first thing to do before the deletion is to verify whether it matches the correct data.
+
+To check that metrics are present in **VictoriaMetrics Cluster** run the following command:
+
+_Warning: the response can return too many metrics, so pleas be careful with series selector_
+
+<div class="with-copy" markdown="1">
+
+```console
+curl -s 'http://127.0.0.1:8481/select/0/prometheus/api/v1/series?match[]=process_cpu_cores_available' | jq
+```
+
+</div>
+
+The expected output:
+
+```json
+{
+  "status": "success",
+  "isPartial": false,
+  "data": [
+    {
+      "__name__": "process_cpu_cores_available",
+      "job": "vmagent",
+      "instance": "vmagent:8429"
+    },
+    {
+      "__name__": "process_cpu_cores_available",
+      "job": "vmalert",
+      "instance": "vmalert:8880"
+    },
+    {
+      "__name__": "process_cpu_cores_available",
+      "job": "vminsert",
+      "instance": "vminsert:8480"
+    },
+    {
+      "__name__": "process_cpu_cores_available",
+      "job": "vmselect",
+      "instance": "vmselect:8481"
+    },
+    {
+      "__name__": "process_cpu_cores_available",
+      "job": "vmstorage",
+      "instance": "vmstorage-1:8482"
+    },
+    {
+      "__name__": "process_cpu_cores_available",
+      "job": "vmstorage",
+      "instance": "vmstorage-2:8482"
+    }
+  ]
+}
+
+```
+
+When you're sure with [time series selector](https://prometheus.io/docs/prometheus/latest/querying/basics/#time-series-selectors) correctness, send a POST request to [delete API](https://docs.victoriametrics.com/url-examples.html#apiv1admintsdbdelete_series) with [`match[]=<time-series-selector>`](https://prometheus.io/docs/prometheus/latest/querying/basics/#time-series-selectors) argument. For example:
+
+**Single-node VictoriaMetrics**:
+
+<div class="with-copy" markdown="1">
+
+```console
+
+curl -v http://127.0.0.1:8428/api/v1/admin/tsdb/delete_series?match[]=process_cpu_cores_available'
+```
+
+</div>
+
+**Cluster version of VictoriaMetrics**:
+<div class="with-copy" markdown="1">
+
+```console
+curl -s 'http://127.0.0.1:8481/select/0/prometheus/api/v1/series?match[]=process_cpu_cores_available'
```

Comment:
I'm not sure about this examples... I see two problems here:
1. You duplicate commands for single and cluster versions. But in prev example it was only for cluster version. This makes example inconsistent or too big. Should we point to https://docs.victoriametrics.com/url-examples.html instead?
2. The address always points to `localhost` - it makes not clear to which `component` user needs to send the request. In [examples](https://docs.victoriametrics.com/url-examples.html#apiv1export) it is clear.